### ISSUE TITLE
Volume / Mute resets on new playback

### ISSUE THREAD

**Issue by Symbai** [unosquare/ffmediaelement#114]
The volume / mute cannot be changed until a video has been started. Moreover if you mute / change volume and then play another file the volume / mute state is reset. 

Anyway to fix either one of the issues or both?

**Comment by mariodivece** [comment]
Correct. The state gets reset every time a new media stream is loaded. I need to change this so the state is kept when new media gets loaded.
